locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/InstallPackagesCustomizer.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.brooklyn.location.jclouds;
```

Comment:
@andreaturli I think so, but the problem is that the `JcloudsLocationCustomizer` underlying it uses hooks that only exist in `JcloudsLocation` so that would need fixed. Maybe just use a plain `EntityInitializer` or, change completely to do things the way `CreateUserPolicy` works instead?